.gitignore

```diff
@@ -10,6 +10,7 @@ iTerm2.xcodeproj/project.xcworkspace
 build/
 .DS_Store
 iTerm.xcodeproj/*
```

Comment:
Can you do this instead:
1. Remove `iTerm.xcodeproj/*`
2. Instead of adding `iTerm2.xcodeproj/*` add `iTerm2.xcodeproj/xcusersdata/*`